docs/src/mpc/outputs.md

```diff
@@ -0,0 +1,40 @@
+# MPC Outputs
+```@docs
+REopt.mpc_results
+```
+
+## MPC ElectricStorage outputs
+```@docs
+REopt.add_electric_storage_results(::JuMP.AbstractModel, ::MPCInputs, ::Dict, ::String)
+```
+
+## MPC HotThermalStorage outputs
+```@docs
+REopt.add_hot_storage_results(::JuMP.AbstractModel, ::MPCInputs, ::Dict, ::String)
```

Comment:
@zolanaj Just a note that these don't currently exist, but keeping them here as placeholders for when mpc results are added for hot and cold storage